content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -2248,18 +2308,31 @@ The _name_ of the `fields` argument on the `@@index` attribute can be omitted:
 @@index([title, author])
 ```
 
+**Note:** In version 3.5.0 and later, new `length` and `sort` arguments are available as a Preview feature. See [Index configuration](../../../../concepts/components/prisma-schema/index-configuration) for more details.
+
+| Name     | Required | Type     | Description                                                                                            |
+| -------- | -------- | -------- | ------------------------------------------------------------------------------------------------------ |
+| `length` | **No**   | `number` | **MySQL only.** Allows you to specify a maximum length for the subpart of the value to be indexed.     |
+| `sort`   | **No**   | `String` | Allows you to specify in what order the entries of the index or constraint are stored in the database. |
+
 #### Signature
 
 ```prisma no-lines
 @@index(_ fields: FieldReference[], map: String?)
 ```
 
-> **Note**: Until version 3.0.0, the signiture was:
+> **Note**: Until version 3.0.0, the signature was:
 >
 > ```prisma no-lines
 > @@index(_ fields: FieldReference[], name: String?)
 > ```
 
+> **Note**: In version 3.5.0 and later, the signature for the `"extendedIndexes"` Preview feature is:
+>
+> ```prisma no-lines
+> @id(_ fields: FieldReference[], map: String?, length: number?, sort: String?)
```

Comment:
Same as above, this is still @id instead of @@index and the length and sort need to go on the field level